package.json

```diff
@@ -53,7 +53,7 @@
     "serverless-plugin-warmup": "^8.2.1",
     "serverless-s3-bucket-helper": "github:Enterprise-CMCS/serverless-s3-bucket-helper#master",
     "serverless-s3-sync": "2.0.0",
-    "serverless-stack-termination-protection": "^1.0.4",
+    "serverless-stack-termination-protection": "^2.0.2",
```

Comment:
this is the only package upgrade. The [changelog](https://github.com/miguel-a-calles-mba/serverless-stack-termination-protection/releases) doesn't list any breaking changes. I updated it to match the version that was in services/database so that I could remove that one